Makefile

```diff
@@ -187,6 +192,27 @@ fmt-local: ## Reformat files using `go fmt`
 imports-local: ## Optimize imports
 	goimports -w -l $$($(FILES_TO_CHECK))
 
+########## Kyma CLI ###########
+KYMA_STABILITY ?= unstable
+
+# $(call os_error, os-type, os-architecture)
+define os_error
+$(error Error: unsuported platform OS_TYPE:$1, OS_ARCH:$2; to mitigate this problem set variable KYMA with absolute path to kyma-cli binary compatible with your operating system and architecture)
+endef
+
+KYMA_FILE_NAME ?= $(shell ./hack/get_kyma_file_name.sh ${OS_TYPE} ${OS_ARCH})
+KYMA ?= $(LOCALBIN)/kyma-$(KYMA_STABILITY)
+
+.PHONY: kyma
```

Comment:
Why have this here in this makefile instead of the `hack/ci/Makefile`? We might need it outside of the github action?